libraries/common/io/files.effekt

```diff
@@ -124,9 +74,9 @@ def readFile(path: String): String / { AsyncIO, Exception[IOError] } = {
 /**
  * Writes the (utf8 encoded) string `contents` into the specified file.
  */
-def writeFile(path: String, contents: String): Unit / { AsyncIO, Exception[IOError] } = {
-  val fd = openFile(path, Write());
-  with on[IOError].finalize { closeFile(fd) }
+def writeFile(path: String, contents: String): Unit / Exception[IOError] = {
+  val fd = direct::open(path, WriteOnly());
```

Comment:
Is this really intended to be `direct::open`? It looks like `direct::open` doesn't check results while  `open` does.

Reply:
That's a mistake, fixed.